scripts/build_python.sh

```diff
@@ -4,11 +4,19 @@ shopt -s inherit_errexit
 
 py_version="${1}"
 
-# Install Python interpreter under e.g. /snekbin/python/3.11/ (no patch version).
+# Install Python interpreter under e.g. /snekbin/python/3.13/ (no patch version)
+# By dropping everything after, and including, the last period.
+install_path="${1%[-.]*}"
```

Comment:
Nit: use `py_version` instead of `1`

Reply:
https://github.com/python-discord/snekbox/pull/218/commits/73e12c84ecb8e3ece9a266f40cee96264efee324